tests/test-cases/Group23-VIC-Machine-Service/TestCases.md

```diff
@@ -0,0 +1,26 @@
+Group 23 - VIC Machine Service
+=======
+
+
+[Test 23-01 - VCH Version](23-01-VCH-Version.md)
```

Comment:
This should just be "Version" (not "VCH Version") since it's the version of the service, not the VCH.